### PR TITLE
feat: add option to use sensor data qos

### PR DESCRIPTION
**Pull request by wep21**
Signed-off-by: wep21 <border_goldenmarket@yahoo.co.jp>

### CODE REVIEW

**Comment by wep21** [comment]
@jbohren @clydemcqueen Could you review this PR? This PR adds the option to use sensor data qos for camera topic.

**Comment by clydemcqueen** [comment]
LGTM.

Can you add this parameter to the README?

I was hoping that we could use the new [externally-configurable QoS options](https://docs.ros.org/en/crystal/Releases/Release-Galactic-Geochelone.html#externally-configure-qos-at-start-up), but:
1. the new options are not supported by image_transport, and
2. the new options are not supported in Foxy.

So for now this simple solution is the best.

I tested this w/ rgb8 (the default) in Foxy, Galactic and Rolling. It should work w/ the jpeg pathway as well.

**Comment by wep21** [comment]
> Can you add this parameter to the README?

@clydemcqueen I added the description in https://github.com/ros-drivers/gscam/pull/79/commits/496b53bf0973b781472a36c61ebcb4317fcb31d6.

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Note that GStreamer is licensed under the LGPL, and GStreamer plugins have their
 * `frame_id`: The [tf2](https://index.ros.org/p/tf2/) frame ID
 * `reopen_on_eof`: Re-open the stream if it ends (EOF)
 * `sync_sink`: Synchronize the app sink (sometimes setting this to `false` can resolve problems with sub-par framerates)
+* `use_sensor_data_qos`: The flag to use sensor data qos for cameara topic(image, camera_info)
```
Comment (wep21) [inline README.md]:
```suggestion
* `use_sensor_data_qos`: The flag to use sensor data qos for camera topic(image, camera_info)
```

**Comment by clydemcqueen** [comment]
Great, thanks!

**Comment by wep21** [comment]
@jbohren friendly ping

**Pull request merged**